style.css

```diff
@@ -116,12 +162,44 @@ article h1 {
   margin: 0;
 }
 
+.about-me li {
+  margin: 0 10px 0 0;
+}
+
 .social-links ul {
   list-style: none;
   display: flex;
   padding-right: 10px;
 }
 
+.frame2 li {
+  margin-left: 30px;
+  font-family: 'Poppins' Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.tags li {
+  color: #3b4eff;
+  font: 500 12px;
+  padding: 0 5px 0 10px;
+  background: #ebebff;
```

Comment:
- [ ] You have done great work on this css file but I think that what the previous reviewer meant was that you change the padding to look like this:

```
padding: 5px 10px;
```